docs/wiki/[Variables]-disable_telemetry.md

```diff
@@ -0,0 +1,32 @@
+## Overview
```

Comment:
Do we want to update this with more context similar to the following page from our ALZ-Bicep Wiki?

https://github.com/Azure/ALZ-Bicep/wiki/CustomerUsage

Or duplicate this page for our Wiki with the necessary updates?

Reply:
Updated in latest commit - see what you think